.travis.yml

```diff
@@ -1,4 +1,13 @@
 sudo: required
+language: node_js
+node_js:
+ - "8"
```

Comment:
There is a bug on the compare screenshot process forcing us to use Node 8 instead of Node 9 which is the latest release.